specification/specs/x_pack/ilm/delete_lifecycle/DeleteLifecycleRequest.ts

```diff
@@ -24,7 +24,8 @@
  */
 interface DeleteLifecycleRequest extends RequestBase {
   path_parts?: {
-    policy: Name
+    policy?: Name
+    policy_id: PolicyId
```

Comment:
I see why you created the alias `PolicyId`, but we should use `Id`. The problem is that `Id` can either be a string or a number when it should always be a number, but our validation infrastructure uses numbers as well.
We should fix this there.

Reply:
I removed the notion of `PolicyId` and exchanged with `Id`